src/index.ts

```diff
@@ -3,11 +3,15 @@ export { Button } from './button/button'
 export { FullscreenLoading } from './fullscreen-loading/fullscreen-loading'
 export { IconButton } from './icon-button/icon-button'
 export { Input } from './input/input'
-export { RequesterInfo } from './requester-info/requester-info'
+export { UserInfo } from './user-info/user-info'
+export { RequestItem } from './request-item/request-item'
+export { MarketplaceItem } from './marketplace-list-item/marketplace-list-item'
```

Comment:
I kind of like keeping these sorted alphabetically for now, as it makes it easy to check if all components were correctly exported. What do you think?

Reply:
Done. Also created issue to introduce a linter rule.